bubble-wrap-style.yaml

```diff
@@ -3876,6 +3085,29 @@ layers:
                         style: italic
                         size: 10px
                         stroke: { color: global.text_stroke_address, width: 4 }
+        entrance-labels:
+            filter:
```

Comment:
This *might* be clearer as:

```
filter:
  all:
    - $zoom: { min: 20 }
    - kind: entrance
    - function() { return global.text_visible_address; }
```

Reply:
While `all` is usually implied, the `function` requires some alternate syntax here.